packages/styled-components/src/sheet/Rehydration.js

```diff
@@ -52,7 +52,7 @@ const rehydrateNamesFromContent = (sheet: Sheet, id: string, content: string) =>
 };
 
 const rehydrateSheetFromTag = (sheet: Sheet, style: HTMLStyleElement) => {
-  const parts = style.innerHTML.split(SPLITTER);
+  const parts = (style.innerHTML || '').split(SPLITTER);
```

Comment:
This change seems unrelated?

Reply:
it isn't really super unrelated. The original issue said that the main problem is that this returns undefined for empty elements in IE11. Basically, I'm just afraid that IE11 users could still experience the bug itself when some HTML snippet is generated using an old version of the library. Basically, just wanted to cover this on both ends

Reply:
I think `.innerHTML` will always return an empty string though